protogen/docs/GRPC.tmpl

```diff
@@ -0,0 +1,97 @@
+---
+{{- $package := (index .Files 0).Package }}
+title: "{{ $package }}"
+url: /grpc_apis/{{ $package }}
+date: {{ now | date "2006-01-02T15:04:05Z07:00" }}
+weight: 50
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/grpc_apis/{{ $package | replace "." "/" }}
+geekdocFilePath: grpc.md
```

Comment:
we should remove that since it doesn't link to a valid target

Reply:
I'll also remove the geekdocEditPath.